src/Magento/FunctionalTestingFramework/Module/MagentoWebDriver.php

```diff
@@ -1080,4 +1082,25 @@ private function executeCronjobs($cronGroups, $timeout, $arguments): string
 
         return sprintf('%s (wait: %ss, execution: %ss)', $cronResult, $waitFor, round($timeEnd - $timeStart, 2));
     }
+
+    /**
+     * Switch to another frame on the page by name, ID, CSS or XPath.
+     *
+     * @param string|null $locator
+     * @return void
+     * @throws \Exception
+     */
+    public function switchToIFrame($locator = null)
```

Comment:
This is a backwards compatible workaround to a new codeception version right?

Reply:
Correct, switchToIFrame of PHPUnit9 takes in a name, css or xpath. Looks like it dropped support for Id. Overriding to support Id since a lot of tests find by Id.